foundry/foundry.py

```diff
@@ -380,18 +381,19 @@ def download(self, globus=True, verbose=False, **kwargs):
             crawl_url = f"{xtract_base_url}/crawl"
             if verbose:
                 print(f"Crawl URL is : {crawl_url}")
+
+            first_ep_dict = {
+                "repo_type": "GLOBUS",
+                "eid": source_ep_id,
+                "dir_paths": [folder_to_crawl],
+                "grouper": grouper,
+            }
+            tokens = {"Transfer": transfer_token, "Authorization": funcx_token}
             crawl_req = requests.post(
-                crawl_url,
-                json={
-                    "repo_type": "GLOBUS",
-                    "eid": source_ep_id,
-                    "dir_path": folder_to_crawl,
-                    "Transfer": transfer_token,
-                    "Authorization": funcx_token,
-                    "grouper": grouper,
-                    "https_info": {"base_url": base_url},
-                },
+                f"{xtract_base_url}/crawl",
+                json={"endpoints": [first_ep_dict], "tokens": tokens},
```

Comment:
I'm surprised the EP query is described as an "endpoint", could you clarify this for my understanding?

Reply:
Maybe we are overloading the term endpoint. Endpoint can be used to describe where a query is being sent to an API, but also can refer to a Globus endpoint. Here we are sending the JSON payload in the variable `json` to the `crawl` _API endpoint_ which requires knowing which _Globus endpoint_ to crawl. Does that help clarify? Pretty confusing!

Reply:
yes that makes more sense! do you think this would warrant an explanatory comment? for future devs